pdk/1.0/modules/ROOT/pages/policies-pdk-apply-policies.adoc

```diff
@@ -0,0 +1,42 @@
+= Applying Custom Policies
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+Applying custom policies is the same as applying xref:gateway::policies-included-directory.adoc[included policies].
+
+When applying custom policies, the configuration parameters are the parameters defined in `definition/gcl.yaml`.
+
+To learn more about defining configuration parameters, see xref:policies-pdk-create-schema-definition.adoc[].
+
+== Before You Begin
+
+* xref:policies-pdk-publish-policies.adoc[Upload your Custom Policy to Exchange]
+
+== Connected Mode
+
+When you apply a custom policy in Connected Mode, the configuration parameters and metadata defined in `gcl.yaml` appear in the API Manager UI.
+
+To apply a custom policy in Connected Mode, see xref:gateway::policies-included-apply.adoc[Apply a Policy in Connected Mode]. 
+
+== Local Mode
+
+For Flex Gateway running in Local Mode, you apply API policies using xref:gateway::flex-local-configuration-reference-guide.adoc[local declarative configuration files]. You specify the location of these configuration files when you first run Flex Gateway.
```

Comment:
```suggestion
For Flex Gateway running in Local Mode, you apply custom policies using xref:gateway::flex-local-configuration-reference-guide.adoc[local declarative configuration files]. You specify the location of these configuration files when you first run Flex Gateway.
```
Should probably be consistent with other usages. Or perhaps omit that word if you want to convey that all policies, custom and non-custom, are applied via the config files.